.mocharc.json

```diff
@@ -6,5 +6,5 @@
   "watch-files": ["src", "test"],
```

Comment:
it's a shame we don't have the 'require a WI PR validation check' in this repo.  While annoying at times it's nice to have that consistency of always having an Issue or WI associated with the PR.